File: frontend/src/app/models/word.ts
import { BaseWord } from "./base-word";
import { GenericLanguageDescriptions } from "./generic-language-descriptions";
import { LinguisticCounter } from "./linguistic-counter";

export interface Word extends BaseWord {    
    identifier: "Word";
    definition: string;
    example: string;
    synonyms: string[];
    linguisticCounter: LinguisticCounter;
    genericLanguageDescriptions: GenericLanguageDescriptions;
}

File: backend/models/Word.py
from .BaseWord import BaseWord

class Word(BaseWord):           
    def __init__(self):
        super().__init__()                
        self.definition = ''
        self.example = ''        
        self.LinguisticCounter = None
        self.LanguageDescriptions = None
        self.synonyms = ''        
            